pkg/authorization/authorizer/authorizer.go

```diff
@@ -267,8 +273,16 @@ func (a openshiftAuthorizationAttributes) verbMatches(verbs util.StringSet) bool
 	return verbs.Has(authorizationapi.VerbAll) || verbs.Has(strings.ToLower(a.GetVerb()))
 }
 
-func (a openshiftAuthorizationAttributes) resourceMatches(resourceNames util.StringSet) bool {
-	return resourceNames.Has(authorizationapi.ResourceAll) || resourceNames.Has(strings.ToLower(a.GetResource()))
+func (a openshiftAuthorizationAttributes) resourceMatches(allowedResources util.StringSet) bool {
+	return allowedResources.Has(authorizationapi.ResourceAll) || allowedResources.Has(strings.ToLower(a.GetResource()))
+}
+
+func (a openshiftAuthorizationAttributes) nameMatches(allowedResourceNames util.StringSet) bool {
+	if len(allowedResourceNames) == 0 {
+		return true
+	}
+
+	return allowedResourceNames.Has(strings.ToLower(a.GetResourceName()))
```

Comment:
I thought they were case-sensitive as well. If they are, we shouldn't `ToLower()` it